pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/index.tsx

```diff
@@ -162,6 +163,11 @@ const dashboards: { [key: string]: GraphDashboard } = {
     component: crossClusterReplicationDashboard,
     isKvDashboard: true,
   },
+  logicalDataReplication: {
+    label: "Logical Data Replication",
+    component: logicalDataReplicationDashboard,
+    isKvDashboard: true,
```

Comment:
Don't quote me on this, but I think this can be false.

Reply:
> I think this can be false
> \- Steven Danna 06/2024